catalog.html

```diff
@@ -0,0 +1,294 @@
+                <span class="control-mark"></span>
+                <span class="control-item">Гостиница</span>
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input type="radio" class="control-input" name="hotel-type" value="motel">
+                <span class="control-mark"></span>
+                <span class="control-item">Мотель</span>
+              </label>
+            </li>
+            <li class="catalog-filter-item">
+              <label class="control">
+                <input type="radio" class="control-input" name="hotel-type" value="apartments">
+                <span class="control-mark"></span>
+                <span class="control-item">Апартаменты</span>
+              </label>
+            </li>
+          </ul>
+        </fieldset>
+
+        <fieldset class="catalog-filter-group">
+          <legend class="catalog-filter-title">Стоимость, ₽:</legend>
+          <label class="catalog-filter-label">
+            <input class="catalog-filter-input" type="number" name="min-price" value="0"> от
+          </label>
+
+          <label class="catalog-filter-label">
+            <input class="catalog-filter-input" type="number" name="max-price" value="9000"> до
+          </label>
+        </fieldset>
+        <button class="button" type="submit">Применить</button>
+        <label>
+          <input type="reset" value="Сбросить">
+        </label>
+      </form>
+
+      <form class="sorting-type">
+        <h3>Найдено гостиниц: 38</h3>
+        <label class="visually-hidden" for="sorting">Сортировка</label>
+        <select class="select-control" id="sorting" name="sorting">
+          <option value="cheap">Сначала дешёвые</option>
+          <option value="popular">Сначала популярные</option>
+          <option value="expensive">Сначала дорогие</option>
+        </select>
+        <button class="visually-hidden" type="submit">Отсортировать гостиницы</button>
+      </form>
+      <a href="#" class="button button-light">
+        <span class="visually-hidden">Показать карточки</span>
+      </a>
+      <a href="#" class="button">
+        <span class="visually-hidden">Показать таблицу</span>
+      </a>
+      <a href="#" class="button">
+        <span class="visually-hidden">Показать список</span>
+      </a>
+
+      <div class="hotel-card">
+        <a class="hotel-card-link" href="#">
+          <img class="hotel-card-image" src="" alt="Гостиница «Amara Resort & Spa»">
+          <h4 class="hotel-card-title">Amara Resort & Spa</h4>
+        </a>
+        <p class="hotel-card-type">Гостиница</p>
+        <p class="hotel-card-price">от 4000 ₽</p>
+        <p class="visually-hidden hotel-stars-mark">4 звезды</p>
+        <p class="hotel-rating">Рейтинг: 8,5</p>
+        <a href="#" class="hotel-card-button button">Подробнее</a>
+        <batton class="favourites-batton" type="submit">В Избранное</batton>
```

Comment:
Что за тег такой `<batton>` ))
```suggestion
        <button class="favourites-batton" type="submit">В Избранное</button>
```

https://github.com/htmlacademy-htmlcss/1902231-sedona-32/blob/1abddaa0ed50312b08051247724032eeee27b489/catalog.html#L178
https://github.com/htmlacademy-htmlcss/1902231-sedona-32/blob/1abddaa0ed50312b08051247724032eeee27b489/catalog.html#L191
https://github.com/htmlacademy-htmlcss/1902231-sedona-32/blob/1abddaa0ed50312b08051247724032eeee27b489/catalog.html#L204
https://github.com/htmlacademy-htmlcss/1902231-sedona-32/blob/1abddaa0ed50312b08051247724032eeee27b489/catalog.html#L217
https://github.com/htmlacademy-htmlcss/1902231-sedona-32/blob/1abddaa0ed50312b08051247724032eeee27b489/catalog.html#L220